pages/api/Using_with_nmap.md

```diff
@@ -28,15 +28,16 @@ This is the shape of the expected payload.
  "nmap_output": {
```

Comment:
In this guide we could consider add a callout 'tip' about using `tidal sync nmap --json`, if they prefer, as it is very similar. Doesn't have to be in this PR.

Having this guide is still very useful, for people doing direct integrations with the API.

Reply:
totally, thanks for bringing it up, let's use that new flag Petr added 😄 .

Reply:
Which one do you think looks better, using a "tip" or a "callout"

#### Tip
![image](https://user-images.githubusercontent.com/22829270/101171402-12f29400-3616-11eb-836c-faff8910fd8e.png)

#### Callout (primary)
![image](https://user-images.githubusercontent.com/22829270/101171571-551bd580-3616-11eb-93ed-2837f4208305.png)

#### Callout 
![image](https://user-images.githubusercontent.com/22829270/101171729-8f857280-3616-11eb-9447-89fdcccdbf18.png)

Reply:
`tip` certainly catches the eye and makes hard to miss the message. if that is the intent you want there, then I vote that one.

Reply:
I'm on the brink of feeling it is too catchy.  🤔 

how about a note?

![image](https://user-images.githubusercontent.com/22829270/101173398-dd9b7580-3618-11eb-9f04-150db9eb18d9.png)

Reply:
nice, looks great. Blue is more calming than the green 😄